velox/dwio/common/IntDecoder.h

```diff
@@ -300,11 +302,138 @@ FOLLY_ALWAYS_INLINE uint64_t IntDecoder<isSigned>::readVuLong() {
   }
 }
 
+template <bool isSigned>
+FOLLY_ALWAYS_INLINE uint128_t IntDecoder<isSigned>::readVuInt128() {
+  if (LIKELY(bufferEnd - bufferStart >= 19)) { // MaxVarintLength128
```

Comment:
19 is magic number?

Reply:
Fixed. Used Varint::kMaxSize128 instead.